Wanna Fly/map.xml

```diff
@@ -140,12 +144,26 @@
         <cuboid min="23,11,42" max="27,14,46"/>
       </region>
     </portal>
+    <portal x="@25" y="@75" z="@12">
+      <region>
+        <cuboid min="23,24,10" max="27,26,14"/>
+      </region>
+    </portal>
   </portals>
   <regions>
+    <union id="portals">
+      <cuboid min="-32,50,68" max="-31,52,69"/>
+      <cuboid min="-9,11,10" max="-5,14,14"/>
+      <cuboid min="23,11,-22" max="27,14,-18"/>
+      <cuboid min="55,11,10" max="59,14,14"/>
+      <cuboid min="23,11,42" max="27,14,46"/>
+      <cuboid min="23,24,10" max="27,26,14"/>
+    </union>
     <apply block-break="never" region="everywhere"/>
+    <apply kit="tp" region="portals"/>
   </regions>
   <broadcasts>
-    <tip after="1s">Hey guys! You can fly like a creative mode in this map! Try it!</tip>
+    <tip after="0s">Hey guys! You can fly like in a creative mode! Try it!</tip>

```

Comment:
```suggestion
    <tip after="0s">This map has flying enabled, try it out!</tip>

```

`in a creative mode!` would be incorrect, posted a suggestion above.

Reply:
Is also rule message incorrect?